Tests/Functional/Fixtures/Indexing/IndexTcaTable.xml

```diff
@@ -60,4 +60,43 @@
         <filelink_sorting>0</filelink_sorting>
     </tt_content>
 
+    <tt_content>
+        <uid>9</uid>
+        <pid>1</pid>
+        <tstamp>1480686370</tstamp>
+        <crdate>1480686370</crdate>
+        <hidden>0</hidden>
+        <sorting>72</sorting>
+        <CType>div</CType>
+        <header>not indexed due to ctype</header>
+        <bodytext>this is the content of header content element that should not get indexed</bodytext>
```

Comment:
Nothing to fix in test, as this content element is NOT indexed due to the CType, while the header element is indexed.

But of course I've fixed the content element type in bodytext.